### DISCUSSION TITLE
how can i remove main search

### DISCUSSION THREAD

**Discussion by FrostTorrents** [jagrosh/MusicBot#799] [Questions]
i am trying to remove the main search function besides youtube, as if you put a ip logger as a link you will get your servers ip logged.

**Comment by MichailiK** [comment]
That's currently not possible, though it'd be something I would look into in the near future.

As a workaround, you can set up your Firewall to only allow outgoing connections to Discord & YouTube.

**Reply by FrostTorrents** [reply]
how would i set a firewall, sorry just got into this discord bot stuff :)

**Reply by MichailiK** [reply]
Firewalls aren't a Discord thing. It heavily varies from your operating system & which firewall you have installed, if any. There isn't much of a "simple guide" for all Firewalls.
